KEYCLOAK-9000 Update stress-testing script

tkyjovsk opened the pull request.

tkyjovsk committed:
KEYCLOAK-9000 Update stress-testing script

hmlnarik commented on testsuite/performance/stress-test.sh:
```diff
@@ -84,23 +122,31 @@ case "${algorithm}" in
-            if [[ `echo "$intervalSize < 0" | bc`           == 1 ]]; then echo "ERROR: Invalid state: lowPoint > highPoint. Stopping the loop."; exit 1; fi
-            ITERATION_INFO="L: $lowPoint    H: $highPoint   intervalSize: $intervalSize   tolerance: $tolerance"
+            intervalSize=`echo "$upperBound - $lowerBound" | bc`
+            usersPerSec=`echo "$lowerBound + $intervalSize * 0.5" | bc`
```
By default, `bc` peforms integer computation:
```
$ bc <<<'4/3'
1
```
`scale` needs to be set for non-integer results:
$ bc <<<'scale=2 ; 4/3'
1.33
```

tkyjovsk commented:
@hmlnarik I made the changes based on our discussion.
- rewrote teh FP ops to bc<<<"scale=X; .."
- rewrote echo to heredoc except few places due to indentation and readability of algorithms
- consistent variable names: external parameters / env. variables with uppercase, local variables and function names with lowercase + underscores

tkyjovsk commented on testsuite/performance/stress-test.sh:
```diff
@@ -84,34 +129,51 @@ case "${algorithm}" in
-            if [[ $testResult == 0 ]]; then 
-                usersPerSecTop=$usersPerSec
+            if [[ `bc<<<"scale=10; $interval_size < $STRESS_TEST_UPS_RESOLUTION"` == 1 ]]; then echo "INFO: interval_size < STRESS_TEST_UPS_RESOLUTION. Stopping the loop."; break; fi
+            if [[ `bc<<<"scale=10; $interval_size < 0"` == 1 ]]; then echo "ERROR: Invalid state:  > upper_bound. Stopping the loop."; exit 1; fi
```
Message should say "Invalid state: lower_bound > upper_bound".
